lib/pymedphys/_experimental/streamlit/apps/sum_doses.py

```diff
@@ -0,0 +1,241 @@
+"""Copyright (C) 2021 Matthew Jennings
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import copy
+import os
+import pathlib
+from typing import Sequence, Union
```

Comment:
```suggestion
from typing import Sequence, Union, List
```

Reply:
Grab yourself this following extension in VSCode:

https://marketplace.visualstudio.com/items?itemName=ms-pyright.pyright

...we should add it to the recommended extensions list in the repo...

Reply:
Thanks! Will do :)

Reply:
@SimonBiggs, there seems to be a small discrepancy between `pyright` warnings in VS Code and the CI suite. On my local VS Code, I get and error showing `List` is not defined (even though it's in a string version of a type hint!):

![image](https://user-images.githubusercontent.com/19767103/106707821-c9d4b580-6641-11eb-8470-a47719102271.png)

But the CI suite complains that List is unused:

![image](https://user-images.githubusercontent.com/19767103/106707894-e83ab100-6641-11eb-9cc5-18bb02214fda.png)

Reply:
Ahh, see, within a string, it still needs to be imported 🙂. ...but pylint doesn't know about its usage when it is within a string. In that case above, you can either ignore the pylint warning, or only put the quotes around Dataset:

```python
Union[List["pydicom.dataset.Dataset"]]
```

Personally I'd opt for not disabling pylint if it is avoidable, but in this case, happy for you to go either way.

Reply:
Also, if you want to run pylint locally:

```
poetry run pymedphys dev lint
```

Reply:
Thanks! I'll go with the first suggestion.